README.md

```diff
@@ -47,6 +47,33 @@ worker.post('/event/*', analyticsMiddleware);
 More than one middleware can be registered with one worker. You just need to keep in mind that the `request` and `response` objects will be passed to each middleware in the same order that they were registered.
 
 
+## Specifying routes
+
+While registering your middlewares, you are not limited just to predefined paths, you have the choice to specify your routes by using *placeholders*. These placeholders are loosely based on [Express' path strings](http://expressjs.com/guide/routing.html#route-paths).
+
+Currently supported placeholders include:
+
+* Anonymous placeholder: `*`  
+Can accomodate any number of characters (including the empty string).
+* Named placeholder: `:<placeholder-name>`  
+Can accomodate any substring, but doesn't allow the empty string (at least 1 chars).  
+The placeholder name could be any number of alphanumeric characters.
+
+```javascript
+worker.get('*'); // will match any path
+
+worker.get('/*'); // will match /, /foo and /foo/bar among others
+
+worker.get('/:path'); // will match /foo and /foo/bar, but won't match /
+```
+
+You could use the backslash character to escape special placeholders (and specify literal asterisks and/or colon characters in your code). *Note: in JavaScript string literals you must double the backslash to achieve the intended effect.*
```

Comment:
Wow. Double scaping could make things really, really ugly. What about using `#` or `?` as scaping character? Some other else? I would prefer to write `'/foo/#*'` rather tahn `'/foo/\\*'` I choose them because these characters are not allowed in the path so if the user wants to use them as literals inside the URL, they must use the encoded versions: `%23` and `%3f`.

A second opinion here, please? @arcturus @flaki ?